api/tests/routes/pro/get_venues_of_offerer_from_siret.py

```diff
@@ -0,0 +1,117 @@
+import pytest
+
+from pcapi.core import testing
+import pcapi.core.offerers.factories as offerers_factories
+from pcapi.core.testing import assert_num_queries
+
+
+@pytest.mark.usefixtures("db_session")
+class Returns200Test:
+    def test_get_venues_of_offerer_from_siret(self, client):
+        siren = "123123123"
+        offerer = offerers_factories.OffererFactory(siren=siren)
+        user_offerer = offerers_factories.UserOffererFactory(user__email="user.pro@test.com", offerer=offerer)
+
+        first_permanent_venue = offerers_factories.VenueFactory(
+            name="Permanent",
+            managingOfferer=user_offerer.offerer,
+            siret=f"{siren}12321",
+            isPermanent=True,
+            publicName="0 - Permanent",
+        )
+
+        second_permanent_venue = offerers_factories.VenueFactory(
+            name="Permanent 1",
+            managingOfferer=user_offerer.offerer,
+            siret=f"{siren}12323",
+            isPermanent=True,
+            publicName="1 - Permanent",
+        )
+
+        offerers_factories.VenueFactory(
+            name="Not Permanent",
+            managingOfferer=user_offerer.offerer,
+            isPermanent=False,
+            siret=f"{siren}12324",
+        )
+
+        auth_request = client.with_session_auth(email=user_offerer.user.email)
+
+        with assert_num_queries(testing.AUTHENTICATION_QUERIES + 1 + 1):  # retrieve Offerer  # retrieve Venues
+            response = auth_request.get("/venues/siret/%s" % "12312312312332")
```

Comment:
```suggestion
            response = auth_request.get("/venues/siret/%s" % "12312312312323")
```
(et en modifiant les assertions)
Pour que le second lieu remonte en premier dans la liste.

Reply:
@francois-seguin j'ai rajouté un test pour ce cas-là (`test_get_venues_of_offerer_from_siret_match_should_be_first`)